faqs/galaxy/tools_tool_doesn't_recognize_any_input_datasets.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Tool doesn't recognize input datasets
+area: tools
+box_type: tip
+layout: faq
+contributors: [jennaj, Melkeb]
+---
+
+The expected input datatype assignment is explained on the tool form. Review the input select areas and the help section below the Execute button. 
+
+Understanding datatypes [FAQ]({% link faqs/galaxy/datatypes_understanding_datatypes.md %}).
+
+No datasets or collections available? Solutions:
+1. Upload or Copy an appropriate dataset for the input into the active history.
+* To review options for getting data into Galaxy see [this FAQ]({% link faqs/galaxy/#data%20upload %}).
```

Comment:
Could be that this link is failing? This is the error:
`File contains invalid characters or sequences: faqs/galaxy/tools_tool_doesn't_recognize_any_input_datasets.md Valid syntax: {% snippet file.ext param='value' param2='value' %} in /home/runner/work/training-material/training-mateqrial/_layouts/faq-page.html`

What I understand from that is that `#data%20upload` should be a valid `.md` file

Reply:
I'm not sure I understand how to fix this link tried adding `.md` at the end and also tried `dataupload.md` but it's still failing.

Reply:
The link is for the header of an FAQ section, not a specific FAQ.

Do we know how to link that? I wasn't sure and put notes in the prior changes about that.